cishouseholds/merge.py

```diff
@@ -40,10 +47,22 @@ def union_multiple_tables(tables: List[DataFrame]):
         list of objects representing the respective input tables
     """
     merged_df = tables[0]
-
     for i, table_n in enumerate(tables[1:]):
-        merged_df, dfn = prepare_for_union(merged_df, table_n)
-        merged_df = merged_df.unionByName(dfn)
+        merged_df, dfn = prepare_for_union(merged_df, table_n, combine_order=reorder_columns)
+        config = get_config()
+        if config != {}:
+            spark_session = get_or_create_spark_session()
+            storage_config = config["storage"]
+            if i == 0:
+                update_table(merged_df, "unioned_survey_responses", mode_overide="overwrite")
+            else:
+                update_table(dfn, "unioned_survey_responses", mode_overide="append")
+            return spark_session.sql(
+                f"SELECT DISTINCT processed_filename FROM \
```

Comment:
There's a helper function elsewhere to read the contents of an existing table by name.